CHANGELOG.md

```diff
@@ -1,352 +1,48 @@
 # Changelog
 
-## [2.0.0-rc.0](https://github.com/woodpecker-ci/woodpecker/releases/tag/2.0.0-rc.0) - 2023-11-11
+## [2.0.0](https://github.com/woodpecker-ci/woodpecker/releases/tag/2.0.0) - 2023-11-21
 
 ### ❤️ Thanks to all contributors! ❤️
 
-@anbraten, @zc-devs, @pat-s, @xoxys, @lafriks, @silverwind, @6543, @pre-commit-ci[bot], @qwerty287, @renovate[bot], @riczescaran, @J-Ha, @Janik-Haag, @jbiblio, @runephilosof-karnovgroup, @bitethecode, @HamburgerJungeJr, @nitram509, @JohnWalkerx, @OskarsPakers, @Exar04, @dominic-p, @categulario, @mzampetakis, @Timshel, @Denperidge, @tomix1024, @lonix1, @s3lph, @math3vz, @LTek-online, @testwill, @klinux, @pinpox, @hpidcock, @ChewingBever, @azdle, @praneeth-ovckd
+@6543, @qwerty287, @renovate[bot], @pat-s
```

Comment:
should it not keep the other autors too ?

Reply:
https://github.com/woodpecker-ci/plugin-ready-release-go/issues/61

Reply:
works now